.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_CAPTCHA_SECRET_KEY=AAA-AAAAAAAAAAAA_AAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
May as well include `SERVER_HOSTNAME` and `UPLOAD_SERVICE_HOST ` here as well

Reply:
Since these are environment variables set within a local machine, putting them in the .env file won't actually do anything. If we do wish to include them, we must prepend them with REACT_APP_ and replace any use of them with the same variable name (so SERVER_HOSTNAME will become REACT_APP_SERVER_HOSTNAME and should be accessed with process.env.REACT_APP_SERVER_HOSTNAME). I am fine with this approach and can make these changes, as this will also remove any variance in setting these environment variables between machines with different OS.

Reply:
We don't need to prepend environment variables with `REACT_APP_`. This project wasn't set up with create react app.
I just tried renaming the captcha env variable and it still worked, since it's being loaded in `server.js` by calling `require("dotenv").config()`

For using env variables in React it will need to be different, so we can use this webpack plugin https://www.npmjs.com/package/dotenv-webpack

As for this file, it would make it easier for people if they could just copy and rename this file on their machine, and then fill in what they need to (captcha key) and have defaults already set for the rest.

Reply:
Agree, was implementing the changes, however I noticed 2 things. 
1. Having environment variables in .env file, but not part of a local machine resulted in webpack.config.js not picking up on these variables and __API__ and __SERVER__ being undefined. Referencing the env variables directly in Captcha.jsx (with process.env.SERVER_HOSTNAME) resulted in a ReferenceError (in promise). I assume that's why you wish to install the plugin, however
2. Using this plugin also does not pick up on __API__ and __SERVER__ as they are still undefined. See image for some maybe relevant documentation?

![image](https://user-images.githubusercontent.com/41775764/118799719-6580ed80-b8e2-11eb-92be-1ad9be299571.png)

However, referencing process.env.SERVER_HOSTNAME will work in Captcha.jsx, even if __SERVER__ (still undefined) does not. 

Would like to hear your thoughts. Happy to discuss this tomorrow.

Reply:
De-structuring has nothing to do with it. The change needed is to reference `process.env.SERVER_HOSTNAME` directly in the React code, no need to define it in webpack.config anymore